docs/player-support/squeezelite.md

```diff
@@ -1,18 +1,28 @@
 # Squeezelite ![Preview image](../assets/icons/slim-icon.svg){ width=70 align=right }
 
-Music Assistant (partly) emulates a Logitech Media Server and has a full implementation of the Slim Protocol (aka slimproto) in the form of support for [Squeezelite](https://en.wikipedia.org/wiki/Squeezelite) clients.
-This means you can use squeezelite software players, which run on almost any hardware, including your own desktop OS and even ESP32 boards, directly with Music Assistant. There is some support for the original Logitech branded Squeezebox hardware players like the Radio and the Duet.
+Music Assistant partly emulates an Logitech Media Server (which is now maintained as Lyrion Music Server, a.k.a. LMS) and has a full implementation of the Slim Protocol (also known as "SlimProto") in the form of player provider support for [Squeezelite](https://en.wikipedia.org/wiki/Squeezelite) clients.
+
+This means you can use Squeezelite client player software and hardware, directly with Music Assistant, as well as control Music Assistant from those Squeezelite client players, (as well as other slimproto emulators such as SqueezeSlave, SoftSqueeze, and SqueezePlay.
+
+Squeezelite clients are available for and run on almost any hardware, including your own desktop OS, and even ESP32-based devices. There is also some legacy support for [the original Logitech branded Squeezebox series of hardware players, (formerly known as SlimDevices), like example the Squeezebox Duet and the Squeezebox Radio](https://lyrion.org/players-and-controllers/hardware-comparison/).
 
 ## Features
 
-- Squeezelite devices are auto detected in Music Assistant
-- Squeezelite devices will play in sync
-- Any physical control buttons on the device should be supported as long as [flow mode](../faq/tech-info.md/#track-queueing) is not enabled
-- The player settings allow configuration of stereo pairs of speakers
+- Squeezelite client devices are automatically detected in Music Assistant.
+- Squeezelite client devices will play in perfect sync and support sync correction.
+  - Speaker sync group support (i.e. support for permanent grouping players).
+- Squeezelite client devices have support for crossfade and gapless playback.
+- Player settings allow the configuration of stereo pairs of speakers.
+- Individual player DSP settings will be used for playback.
+- Up to 384kHz max at 24bit sample rate for PCM if supported by the client device.
+- Lossless Stream support.
```

Comment:
Actually lets remove from "Squeezelite client devices will play in perfect sync" to here as it is just replicating what is already in the table on the main player provider page.  We could just provide a link back as the last item to the player provider page for the cross platform features? The existing text is from before the summary table was added. Also please remove the fullstop from the end of any bullet list item. The reason for this is that is the standard across all the pages to avoid any unnecessary wrapping of text with long sentences